frontend/build.go

```diff
@@ -59,6 +60,30 @@ func makeRequestHandler(target string) dockerui.RequestHandler {
 	return h
 }
 
+var PassthroughGetters = map[string]func(ocispecs.Platform) string{
+	"TARGETOS": func(p ocispecs.Platform) string {
+		return p.OS
+	},
+	"TARGETARCH": func(p ocispecs.Platform) string {
+		return p.Architecture
+	},
+	"TARGETVARIANT": func(p ocispecs.Platform) string {
+		return p.Variant
+	},
+	"TARGETPLATFORM": func(p ocispecs.Platform) string {
+		return platforms.Format(p)
+	},
```

Comment:
How about `BUIILDOS`, `BUILDARCH`, etc...?

Reply:
Implemented. Weirdly, the docker client stores an array of build platforms, but only one default build platform ever seems to be set, so I am assuming a singular build platform for now